stable/rethinkdb/templates/rethinkdb-cluster-stateful-set.yaml

```diff
@@ -8,16 +8,20 @@ metadata:
     heritage: {{ .Release.Service | quote }}
     release: {{ .Release.Name | quote }}
 spec:
+  selector:
+    matchLabels:
+      app: "{{ template "rethinkdb.name" . }}-cluster"
+      release: {{ .Release.Name | quote }}
   serviceName: "{{ template "rethinkdb.fullname" . }}-cluster"
   replicas: {{ .Values.cluster.replicas }}
+  updateStrategy:
+    type: RollingUpdate
   template:
     metadata:
       name: "{{ template "rethinkdb.fullname" . }}-cluster"
       labels:
         app: "{{ template "rethinkdb.name" . }}-cluster"
-        heritage: {{ .Release.Service | quote }}
         release: {{ .Release.Name | quote }}
-        chart: {{ template "rethinkdb.chart" . }}
```

Comment:
Removing the chart version here because it causes issues whenever you update to a new chart version. I believe the best practice is to make sure these are immutable.

Also, I don't think we need the `heritage` label either. Not even sure what that is fore, tbh...